src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1013,7 +1034,31 @@ private CorInfoType asCorInfoType(CORINFO_CLASS_STRUCT_* cls)
         }
 
         private int appendClassName(short** ppBuf, ref int pnBufLen, CORINFO_CLASS_STRUCT_* cls, bool fNamespace, bool fFullInst, bool fAssembly)
-        { throw new NotImplementedException("appendClassName"); }
+        {
+            // We support enough of this to make SIMD work, but not much else.
+
+            // TODO: figure out why the marshalling for fFullInst and fAssembly is wrong.
```

Comment:
This kinda looks like a bug in marshalling. I couldn't figure out what's wrong with the signature. Suggestions welcome.